textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/source/IndexSearchSourceOperator.java

```diff
@@ -0,0 +1,28 @@
+package edu.uci.ics.textdb.dataflow.source;

```

Comment:
Is it better to include this "IndexSearchSourceOperator.java" in the next pull request to keep the modularity?